src/R/Sql/Impl/Publish/DacPackageServices.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.R.Components.Sql.Publish;
+
+namespace Microsoft.VisualStudio.R.Sql.Publish {
+    [Export(typeof(IDacPackageServices))]
```

Comment:
I actually think `IDacPackageServices` shouldn't been exported, cause otherwise someone can write code that imports `IDacPackageServices` instead of `IDacPackageServicesProvider`.